DB/SharedConstants.lua

```diff
@@ -113,6 +113,7 @@ C.UIMAPIDS = {
 	OHN_AHRAN_PLAINS = 2023,
 	THE_AZURE_SPAN = 2024,
 	THALDRASZUS = 2025,
+	THE_FORBIDDEN_REACH = 2151,
```

Comment:
This should go away automatically after a rebase, since the diff from the updated `master` would be empty.